src/app/parcels/ActionBar/ActionModals/ShoppingListModal.tsx

```diff
@@ -1,49 +1,13 @@
 "use client";
 
 import React, { useState } from "react";
-import GeneralActionModal, {
-    Heading,
-    Paragraph,
-    maxParcelsToShow,
-    ActionModalProps,
-} from "./GeneralActionModal";
+import GeneralActionModal, { ActionModalProps, maxParcelsToShow } from "./GeneralActionModal";
 import SelectedParcelsOverview from "../SelectedParcelsOverview";
-import { ParcelsTableRow } from "../../parcelsTable/types";
-import { StatusType, getStatusErrorMessageWithLogId } from "../Statuses";
+import { getStatusErrorMessageWithLogId } from "../Statuses";
 import ShoppingListPdfButton from "@/pdf/ShoppingList/ShoppingListPdfButton";
 import { ShoppingListPdfError } from "@/pdf/ShoppingList/getShoppingListData";
 import { sendAuditLog } from "@/server/auditLog";
-import { nullPostcodeDisplay } from "@/common/format";
-
-interface ShoppingListsConfirmationProps {
```

Comment:
Ticket asked to add warning to shipping labels and shopping lists - is it just the case that shopping lists had a warning and we only needed it on shipping labels? Does this have the same behaviour as the new method?

Reply:
Yes it turned out the shopping list already had one, but it had slightly different behaviour to what we decided on for this ticket so I shared the new component across both modals.